lte/protos/mconfig/mconfigs.proto

```diff
@@ -468,3 +468,22 @@ message GatewayDNSConfigRecordsItems {
   repeated string cname_record = 3;
   string domain = 4;
 }
+
+//----------------------------------------------------------------------------
+// AgwD configs
+//----------------------------------------------------------------------------
+message AgwD {
+  enum LogLevel {
```

Comment:
there's already a `LogLevel` on `magma/orc8r/protos/common.proto`, but that one does not contain UNSET value, any reason apart from that to not use the common version?

Reply:
Having UNSET is probably a good idea as otherwise we can't tell the difference between DEBUG and the value being empty. (Or can we?) I think we want the default value if not specified to be INFO, probably. But I suppose it is confusing that it digerges from the other protos.

Reply:
Note: in proto3 all enums should have a 0 = UNSET/INVALID value. Using the 0 value as default is not recommended because 1. distinguishing between unset vs. set can be important sometimes, 2. the default value might change and you can't change explicitly set serialized values.

In this particular case, the common proto also defines a FATAL level, which we no longer support.